x11-drivers/nvidia-drivers/nvidia-drivers-460.39.ebuild

```
@@ -236,15 +199,15 @@ donvidia() {
 
 	# If the library has a SONAME and SONAME does not match the library name,
 	# then we need to create a symlink
-	if [[ ${nv_SOVER} ]] && ! [[ "${nv_SOVER}" = "${nv_LIBNAME}" ]]; then
+	if [[ ${nv_SOVER} ]] && ! [[ ${nv_SOVER} == ${nv_LIBNAME} ]]; then
```

Go bash all the way?

```suggestion
	if [[ -n ${nv_SOVER} && ${nv_SOVER} != ${nv_LIBNAME} ]]; then
```